public/js/controllers/traffic_graphs_controller.js

```diff
@@ -1034,7 +1034,7 @@ export class TrafficGraphsController {
             // Calculate time diff bw current and lastUpdateTimeStamp
             let currentTime = new Date(),
                 difference_minutes = (currentTime.getTime() - lastUpdateTimeStamp.getTime()) / 60000;
-            if (difference_minutes > 20) {
+            if (difference_minutes > 30) {
```

Comment:
why are we increasing this? The dashboard updates every 10 mins so the sooner we know it is down the better.

Reply:
I changed to 30 minutes to be same as for system metrics but you are right.I have updated to 20 minutes.